packages/ui/src/lib/tooltip/Tooltip.svelte

```diff
@@ -77,7 +77,7 @@ export let left = false;
     class:bottom-right={bottomRight}>
     {#if tip}
       <div
-        class="inline-block py-2 px-4 rounded-md bg-[var(--pd-tooltip-bg)] text-[var(--pd-tooltip-text)] border-[1px] border-[var(--pd-tooltip-border)]"
+        class="inline-block py-2 px-4 rounded-md bg-[var(--pd-tooltip-bg)] text-[var(--pd-tooltip-text)] border-[1px] border-[var(--pd-tooltip-border)] {$$props.class}"
```

Comment:
@benoitf since `Tooltip.svelte` is also used by components other than `ErrorMessage`, could `$${props.class}` be a problem if it is undefined in other components?

Reply:
no it's 👍